### DISCUSSION TITLE
Monitoring logs in Apisix

### DISCUSSION THREAD

**Discussion by yaswanth507** [apache/apisix#10640] [General]
Can any body let me know we want to get body request and response data, including headers data in apisix logs

**Comment by kayx23** [comment]
See these examples and tweak to your logger & use cases:

https://docs.api7.ai/hub/kafka-logger#log-request-and-response-headers-with-plugin-metadata
https://docs.api7.ai/hub/kafka-logger#log-request-bodies-conditionally
